actors/builtin/power/power_test.go

```diff
@@ -145,6 +148,115 @@ func TestPowerAndPledgeAccounting(t *testing.T) {
 		require.Equal(t, big.Zero(), claim2.RawBytePower)
 		require.Equal(t, big.Zero(), claim2.QualityAdjPower)
 	})
+
+	t.Run("power accounting crossing threshold", func(t *testing.T) {
+		rt := builder.Build(t)
+		actor.constructAndVerify(rt)
+
+		actor.createMinerBasic(rt, owner, owner, miner1)
+		actor.createMinerBasic(rt, owner, owner, miner2)
+		actor.createMinerBasic(rt, owner, owner, miner3)		
+		actor.createMinerBasic(rt, owner, owner, miner4)				
+		actor.createMinerBasic(rt, owner, owner, miner5)
+
+		actor.updateClaimedPower(rt, miner1, smallPowerUnit, mul(smallPowerUnit, 2))
+		actor.updateClaimedPower(rt, miner2, smallPowerUnit, mul(smallPowerUnit, 2))		
+		actor.updateClaimedPower(rt, miner3, smallPowerUnit, mul(smallPowerUnit, 2))				
+
+		actor.updateClaimedPower(rt, miner4, powerUnit, mul(powerUnit, 2))
+		actor.updateClaimedPower(rt, miner5, powerUnit, mul(powerUnit, 2))		
+
+		// Below threshold small miner power is counted
+		expectedTotalBelow := big.Sum(mul(smallPowerUnit, 3), mul(powerUnit, 2))
+		actor.expectTotalPower(rt, expectedTotalBelow, mul(expectedTotalBelow, 2))
+
+		// Above threshold (power.ConsensusMinerMinMiners = 3) small miner power is ignored
+		actor.updateClaimedPower(rt, miner3, powerUnit, mul(powerUnit, 2))
+		expectedTotalAbove := big.Sum(smallPowerUnit, mul(powerUnit, 3))
+		actor.expectTotalPower(rt, expectedTotalAbove, mul(expectedTotalAbove, 2))
+
+		st := getState(rt)
+		assert.Equal(t, int64(3), st.NumMinersMeetingMinPower)
+
+		// Less than 3 miners below threshold again small miner power is counted again
+		negPowerUnit := powerUnit.Neg()
+		actor.updateClaimedPower(rt, miner3, negPowerUnit, mul(negPowerUnit, 2))
+		actor.expectTotalPower(rt, expectedTotalBelow, mul(expectedTotalBelow, 2))
+	})
+
+	t.Run("all miner power dissapears when miner dips below min power threshold", func(t *testing.T) {
```

Comment:
nit: I found this a little confusing (like total power would be zero). Maybe "removes all a miner's power when..."